### PR TITLE
Better route parsing

### PR DESCRIPTION
**Pull request by danpersa**
Instaskip is now able to parse both normalized and denormalized routes

### CODE REVIEW

**Comment by danpersa** [comment]
Should add support for `Host(/^(m|www)-(it|pl)[.]domain[.]net$/)` also

File: src/instaskip/impl/eskip_map_process.clj [hunk 1]
```diff
@@ -5,16 +5,75 @@
...
-  (let [trimmed-hosts (->> hosts-string
-                           (re-find (re-pattern "\\/\\^\\((.*)\\)\\$\\/"))
-                           peek)
-        hosts-with-normalized-dot (.replace trimmed-hosts "[.]" ".")]
+(defn- trim-hosts [hosts-string]
+  (if-some [trimmed-hosts (re-find (re-pattern "\\/\\^(.+)\\$\\/") hosts-string)]
+    (peek trimmed-hosts)
+    hosts-string))
+
+(trim-hosts "/^(www|m)[.]host1[.]com$/")
```
Comment (mo-gr) [inline src/instaskip/impl/eskip_map_process.clj]: why is this here?

Comment (danpersa) [inline src/instaskip/impl/eskip_map_process.clj]: removed
